.ci-config.json

```diff
@@ -1,6 +1,6 @@
 {
```

Comment:
I think when you use pull_request_target this file is not needed and vars can be provided as here: https://github.com/kyma-project/test-infra/blob/main/.github/workflows/pull-plan-prod-terraform.yaml#L47-L48

Reply:
There're was a discussion with @dekiel to stick for it now and discuss it further in the team.